docs/markdown/snippets/templates.md

```diff
@@ -6,3 +6,21 @@ scratch, simply go to an empty directory and type:
 ```meson
 meson init --name=myproject --type=executable --language=c
 ```
+
```

Comment:
There is a project freeze at the moment and this change will not be merged until 0.46.0. Could you please create a separate snippet for this change?

Reply:
Just a separate file in the same directory? Sure, no problem.